integrations/docker/images/chip-build-vscode/Dockerfile

```diff
@@ -110,6 +110,7 @@ ENV TELINK_ZEPHYR_BASE=/opt/telink/zephyrproject/zephyr
 ENV TELINK_ZEPHYR_SDK_DIR=/opt/telink/zephyr-sdk-0.15.2
 ENV TI_SYSCONFIG_ROOT=/opt/ti/sysconfig_1.13.0
 ENV ZEPHYR_BASE=/opt/NordicSemiconductor/nrfconnect/zephyr
+ENV ZEPHYR_SDK_INSTALL_DIR=/opt/NordicSemiconductor/nRF5_tools/zephyr-sdk-0.16.0
```

Comment:
This changes the vscode image in an incompatible way, so it needs a version file update.

Reply:
@andy31415 Actually the `chip-build-vscode` image already imported latest `chip-build-nrf-platform` image (refer to [Dockerfile](https://github.com/project-chip/connectedhomeip/blob/1d9f02da2697b57605050940652b2363e022391f/integrations/docker/images/chip-build-vscode/Dockerfile#L2)), so the `chip-build-vscode` has the latest version ZEPHYR SDK installed.

```
FROM connectedhomeip/chip-build-nrf-platform:${VERSION} AS nrf
...
COPY --from=nrf /opt/NordicSemiconductor/nRF5_tools /opt/NordicSemiconductor/nRF5_tools
```

Reply:
BTW, the Docker `-efr32` build failure is caused by issue #27362 and is nothing to do with current PR.

Reply:
@woody-apple this was not resolved - any changes in docker files should have updated the version. This is obvious here since this "fixes chef" meaning after the update something changed...but you cannot do that without updating the version.

Reply:
My point would be: even if file content is not changed, the env values are part of the image, so the image is changed. If the image changes, the version has to be bumped up, that way if I say "I am using version 0.7.17" I know exactly what I have without saying "pulled today" or "pulled 1 week ago".

Reply:
fixed, ty: https://github.com/project-chip/connectedhomeip/pull/27384